[GLOBAL]
print_exception_stacktrace: True

[python-repos]
repos: ['third_party/', 'https://svn.apache.org/repos/asf/aurora/3rdparty/debian/jessie64/python/']
